### ISSUE TITLE
Ability to programmatically set the open Step

### ISSUE THREAD

**Issue by mariodavid** [mariodavid/cuba-component-wizard#17]
```
As a application developer using the addon
When a particular scenario in the UI occurs, I want to prgrammatically activate a particular wizard step 
so that the user can jump directly from Step 1 to 4 in case a particular option was chosen e.g.
```

**Comment by abdelaziz-allam** [comment]
Hi Mario , 

I want to start coding to implement navigation to specific step using action button inside wizard 

either we can start parallel or you can give me like overview from exactly shall i start and i will download clone base code and start working on it and ask for feedback

**Comment by mariodavid** [comment]
hi,

you can take a look at the component interface: https://github.com/mariodavid/cuba-component-wizard/blob/master/modules/gui/src/de/diedavids/cuba/wizard/gui/components/Wizard.java

This API probably needs a new method `activateStep(WizardStep wizardStep)`. Then the WebWIzard implementation of the interface needs to call the corresponding methods on the TabSheet that it uses under the hood.

I hope this helps to get started.

Cheers
Mario

**Comment by mariodavid** [comment]
hi, 

this feature is done in #23 and released in 0.8.1.

Cheers
Mario
